manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE

```

```suggestion
PackageIdentifier: Ashampoo.BurningStudio.Free
```
After changing this you will also need to update the file path to be /manifests/a/Ashampoo/BurningStudio/Free